build/__nodeVersions.sh

```diff
@@ -13,4 +13,4 @@ NODE16_VERSION='16.19.0'
 NODE18_VERSION='18.14.0'
 NODE_APP_INSIGHTS_SDK_VERSION='1.8.7'
 PM2_VERSION='4.5.6'
-NPM_VERSION='6.14.15'
+NPM_VERSION='9.6.4'
```

Comment:
It would probably be _ok_ to assume to upgrade to the latest lts version but I think it would be good to let them know before we merge these changes to main?